restore/README.md

```diff
@@ -0,0 +1,35 @@
+# Restore Instructions
+
+Please follow these instructions if you are restoring any settings that has been missed or deleted during/after the installation.
+
+# Please use this set of instructions:
+
+a) You have a 4.0.3.1 installed and required to restore any settings.
+b) Already have a gitops-repo configured for ISD (AP and Spinnaker) Configuration.
+
+# Use Case 1: If a namespace is deleted in a cluster
+These instructions can be followed when the ISD installed namespace is deleted by any chance and we need to restore our instance.
+
+1. ISD stores all the configuration in the git repo, we can check for the "values.yaml" file that exists in the directory (root of the gitops-repo) that is already configured with the required parameters.
+2. Use the same "values.yaml" and execute the below commands to install it again.
```

Comment:
Only 4 commands will be present
Git Token need to be applied
Inputcm
service account
Job Apply